dev-lang/nim/nim-1.6.10-r1.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 edo multiprocessing toolchain-funcs xdg-utils
+
+PATCH_PV="1.6.6_p1"
+DESCRIPTION="compiled, garbage-collected systems programming language"
+HOMEPAGE="https://nim-lang.org/"
+SRC_URI="
+	https://nim-lang.org/download/${P}.tar.xz
+	experimental? (
+		https://git.sr.ht/~cyber/${PN}-patches/archive/${PATCH_PV}.tar.gz
+			-> nim-patches-${PATCH_PV}.tar.gz
+	)
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="debug doc experimental +test-js +test-valgrind test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	test? (
+		dev-db/sqlite:3
+		dev-libs/boehm-gc
+		dev-libs/libffi
+		dev-libs/libpcre:3
+		dev-libs/openssl
+		media-libs/libsdl
+		media-libs/libsfml
+	)
+"
+
+BDEPEND="
+	sys-process/parallel
```

Comment:
So parallel works now? Does it still spam the build log with ads?

Reply:
I touch `will-cite` file so it doesn't spam the logs